### PR TITLE
Add WriteInPublic API client

### PR DESCRIPTION
**Pull request by chrismytton**
This is a very simple wrapper around the WriteInPublic API. Currently it only implements the features that we need for integrating it into Pombola.

I've added some tests for the basic functionality, but in the interests of time I've skipped writing tests for the `.people()` and `.answers()` methods, so will need to revisit that at some point in the future.

Extracted from #2329 
Part of #1682

### CODE REVIEW

File: pombola/writeinpublic/client.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            'author_email': author_email,
+            'subject': subject,
+            'content': content,
+            'writeitinstance': "/api/v1/instance/{}/".format(self.instance_id),
+            'persons': persons,
+        }
+        try:
+            return requests.post(url, json=payload, params=params)
+        except requests.exceptions.RequestException as err:
+            raise self.WriteInPublicException(err)
```
Comment (mhl) [inline pombola/writeinpublic/client.py]: The argument to this exceptions constructor is expected to be a string, I think, so perhaps this should be:
```
raise self.WriteInPublicException(unicode(err))
```

File: pombola/writeinpublic/client.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    def get_messages(self, person_popolo_uri):
+        url = '{url}/api/v1/instance/{instance_id}/messages/'.format(url=self.url, instance_id=self.instance_id)
+        params = {
+            'format': 'json',
+            'username': self.username,
+            'api_key': self.api_key,
+            'person__popolo_uri': person_popolo_uri,
+        }
+        response = requests.get(url, params=params)
```
Comment (mhl) [inline pombola/writeinpublic/client.py]: I think that all calls to `requests.get` or `requests.post` should be followed by `response.raise_for_status()` afterwards, or you won't get an exception for 4xx and 5xx responses, and will get a less helpful error later...

Comment (chrismytton) [reply pombola/writeinpublic/client.py]: Ah good shout, I didn't know about that method, added that in de1f549.

**Comment by chrismytton** [comment]
@mhl I've made those changes now, so I think this is ready for another review.

**Pull request merged**